src/common/snap.F90

```diff
@@ -1898,6 +1900,11 @@ subroutine read_inputfile(snapinput_unit)
           write (error_unit, *) 'WARNING. Too many FIELD INPUT files'
           write (error_unit, *) '  ==> ', cinput(pname_start:pname_end)
         end if
+      case ('field.spinupsteps')
```

Comment:
This is not implemented in the `netCDF` backend, should we compile gate this?

Reply:
implememented in netcdf